indra/assemblers/pysb_assembler.py

```diff
@@ -177,11 +181,11 @@ def get_mod_site_name(mod_type, residue, position):
     name = ('%s%s' % (mod_str, mod_pos))
     return name
 
-def get_activating_mods(agent, agent_set):
-    act_mods = agent_set[agent.name].activating_mods
-    if not act_mods:
-        act_mods = [{}]
-    return act_mods
+def get_active_forms(agent, agent_set):
```

Comment:
Is the intent to eventually implement `get_inactive_forms` too? If so, might be good to implement it as a stub that just throws `NotImplementedError`.